services/core-web/src/components/mine/Projects/ProjectOverviewTab.js

```diff
@@ -44,13 +49,19 @@ export class ProjectOverviewTab extends Component {
                 </Typography.Text>
               )}
               <br />
-              <Typography.Text>{c.name}</Typography.Text>
-              <br />
-              <Typography.Text>{c.phone_number}</Typography.Text>
-              <br />
-              <Typography.Text>
-                <a href={`mailto:${c.email}`}>{c.email}</a>
-              </Typography.Text>
+              {c.is_project_lead_contact && !c.name ? (
+                <Typography.Text>Project Lead has not been assigned</Typography.Text>
+              ) : (
+                <>
+                  <Typography.Text>{c.name}</Typography.Text>
+                  <br />
+                  <Typography.Text>{c.phone_number}</Typography.Text>
```

Comment:
Judging by your first screenshot "phone_number" is not required on Inspectors and results in a blank line in the card. Can we conditionally render any fields that are optional here?

Reply:
My bad, no need for that in phone number, the field is required. Just email field is optional.